change/@uifabric-experiments-2019-07-10-15-32-26-jocobb-tile-meta-hover.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "Add screener tests for media tiles",
```

Comment:
FYI This file can be omitted as its format is only supported in `master`.

Reply:
Thanks for pointing that out, I was not aware. I'll just remove that commit